### ISSUE TITLE
Implement `enrollment` view and template

### ISSUE THREAD

**Issue by angela-tran** [cal-itp/benefits#2244]
Depends on #2284

This view allows the user to enroll their payment card to receive their transit benefit using the Littlepay hosted verification flow.

## Acceptance Criteria

- [x] User is able to enroll their payment card using the Littlepay hosted verification flow

**Comment by angela-tran** [comment]
I was thinking of the success template as a part of this ticket and decided it should be a separate PR so that #2340 wasn't too big.
